source/process/asset-guidelines.rst

```diff
@@ -114,52 +114,49 @@ The following include guidelines for specific elements of an asset.
 Feature Title
 ^^^^^^^^^^^^^^
 
-1. Appears in the bottom right corner of the asset
+1. Appears in the top left corner of the asset
 2. Title should "communicate the benefit" of the work, while referencing its name, followed by "for Mattermost"
   
   - Sometimes the title is a straightforward description of an app that can now connect to Mattermost, like "Jira integration for Mattermost"
   - Sometimes the title needs to be non-standard, like "Gitter integration for Mattermost via Matterbridge"
 
-3. Title should be sentence case
+3. Title should always be title case
 4. Short titles are better than longer titles
 
 Feature Creator
 ^^^^^^^^^^^^^^^^
 
-1. Appears in the bottom left corner of the left "representative art" pane
-2. Displays who created the integration (Twitter or GitHub account of the creator)
-3. If text doesn’t blend well with left pane representative art, adjust the outer glow effect
+1. Appears in the bottom right corner of the asset footer
+2. Text in footer should be in ALL lower case i.e. “by matterhorn-chat”
```

Comment:
This from Ian in [Integrations & Apps](https://pre-release.mattermost.com/core/pl/wtsp8q98j7y8zxnhbf1p1zxm7r) channel:

"I think we're now going with JB's suggestion of by {creator} so I think we keep the 'by' lowercase then {creator} lowercase if it's a github username, but then capital case if it's a company name or a person's name?"